server/src/main/java/tabletop/domain/match/tournament/TournamentProcess.java

```diff
@@ -0,0 +1,43 @@
+package tabletop.domain.match.tournament;
+
+import tabletop.domain.user.User;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Entity
+@Inheritance(strategy = InheritanceType.JOINED)
```

Comment:
no, it can't, Tournament entity has reference to it and jpa doesn't allow polymorphic relationships